### PR TITLE
:pencil2: #API : Create & Modify some functions

### PR DESCRIPTION
**Pull request by K4VK4Z**

### CODE REVIEW

File: src/api/player/server/kickPlayer.lua [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    end
+    CreateThread(function()
+        while (true) do 
+            if (GetPlayerPing(playerSrc) >= 0) then
+                break
+            end
+            Wait(200)
+            DropPlayer(playerSrc, reason)
+        end
+    end)
```
Comment (CsokiHUN) [inline src/api/player/server/kickPlayer.lua:18]: Why need this?
Especially if we dropped the player ?

File: src/api/ped/client/getMugshot.lua [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+function API_Ped:getMugshot(pedId, transparentBackground)
+    if (not (DoesEntityExist(pedId))) then
+        return _NCS:die("Target ped does not exists")
+    end
+    local headshot = RegisterPedheadshot(pedId)
+    if (transparentBackground) then
+        UnregisterPedheadshot(headshot)
+        Wait(100)
+        headshot = RegisterPedheadshotTransparent(pedId)
+    end
```
Comment (CsokiHUN) [inline src/api/ped/client/getMugshot.lua:15]:
```suggestion
    local headshot = (transparentBackground) and RegisterPedheadshotTransparent(pedId) or RegisterPedheadshot(pedId)
```

**Review by CsokiHUN — CHANGES_REQUESTED** [review]

File: src/api/player/server/kickPlayer.lua (hunk 1 above)
Comment (K4VK4Z) [reply src/api/player/server/kickPlayer.lua:18]: This forces the kick of the player in case the server had a latency in the execution of the function

Comment (CsokiHUN) [reply src/api/player/server/kickPlayer.lua:18]: Oh okay. I thinked the fivem client solved this problem.

File: src/api/player/server/kickPlayer.lua [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        DropPlayer(playerSrc, reason)
+    end
+    CreateThread(function()
+        while (true) do 
+            if (GetPlayerPing(playerSrc) >= 0) then
+                break
+            end
+            Wait(200)
+            DropPlayer(playerSrc, reason)
+        end
```
Comment (MoskalykA) [inline src/api/player/server/kickPlayer.lua:17]:
```suggestion
    DropPlayer(playerSrc, (reason or _Literals.KICK_DEFAULT_MESSAGE))
```

File: src/api/player/server/kickPlayer.lua [hunk 4]
```diff
@@ -3,17 +3,6 @@
...
-    CreateThread(function()
-        while (true) do 
-            if (GetPlayerPing(playerSrc) >= 0) then
-                break
-            end
-            Wait(200)
-            DropPlayer(playerSrc, reason)
-        end
+    DropPlayer(playerSrc, (reason or _Literals.KICK_DEFAULT_MESSAGE))
     end)
```
Comment (CsokiHUN) [inline src/api/player/server/kickPlayer.lua:7]: :bug: fix
```suggestion
```

**Review by CsokiHUN — CHANGES_REQUESTED** [review]

**Comment by PABLO-1610** [comment]
image

**Review by MoskalykA — APPROVED** [review]

**Pull request merged**